modules/appeals_api/lib/appeals_api/sidekiq_retry_notifier.rb

```diff
@@ -35,6 +35,12 @@ def request_headers
         }
       end
 
+      def retried_at(retried_time)
+        return 'was not retried' unless retried_time
```

Comment:
bit of a nitpick, but if the flow hits this line, it looks like the message will read like this:
```
This job failed at: {current-time}, and was retried at: was not retried
```
would it make more sense to have this `retried_at` method return the last bit of the above sentence instead? so
```
      def retried_at(retried_time)
        return 'was not retried' unless retried_time
        
        "was retried at: #{Time.zone.at(retried_time)}"
      end
```

i dont see anywhere else this method is called that this change might break at least

Reply:
Yeah, that's better. I'll change it! Thanks